fluent-react/src/localized.ts

```diff
@@ -63,37 +57,37 @@ function Localized(props) {
 
   if (!l10n) {
     // Use the wrapped component as fallback.
-    return child;
+    return createElement(Fragment, null, child);
   }
 
   const bundle = l10n.getBundle(id);
 
   if (bundle === null) {
     // Use the wrapped component as fallback.
-    return child;
+    return createElement(Fragment, null, child);
   }
 
-  const msg = bundle.getMessage(id);
-  const [args, elems] = toArguments(props);
-  let errors = [];
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const msg = bundle.getMessage(id)!;
```

Comment:
I'm probably missing something here. Why is that assertion safe here? (I might not be the only ignorant person here, so maybe a comment is warranted).

Reply:
Good point, I'll add a comment. It's safe because the `bundle` comes from `l10n.getBundle` which makes the `hasMessage` check. The `hasMessage` followed by `getMessage` flow doesn't work too well in typed languages, unfortunately.